portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,145 @@
+                <h3>Technologies</h3>
+                <p>Kotlin, C++, HTML, CSS, JavaScript/TypeScript, Algorithms & Data Structures.</p>
+            </div>
+            <div class="skills__languages">
+                <h3>Languages</h3>
+                <p>Russian(Native), English(B2), Spanish(A1/A2).</p>
+            </div>
+        </section>
+        <section class="cats">
+            <ul>
+                <li class="cat">
+                    <img class="cat_image" src="images/loki.png">
+                    <div class="cat_description">
+                        <h3>Loki</h3>
+                        <p>Cute ball of fur and fat. Lives in Omsk, spends his summer in the country house with
+                        his friends, catches mice, tries to catch a sparrow sometimes, but unsuccessfully.</p>
+                    </div>
+                </li>
+                <li class="cat">
+                    <img src="images/cheburek.png">
+                    <div class="cat_description">
+                        <h3>Cheburek</h3>
+                        <p>The most affectionate and impudent cat in the world. Lives with my sister and Belyash. 
+                        Likes watching TV, sit on people's lap and eat.</p>
+                    </div>
+                </li>
+                <li class="cat">
+                    <img class="cat_image" src="images/belyash.png">
+                    <div class="cat_description">
+                        <h3>Belyash</h3>
+                        <p>Arrogant and handsome. Lives with my sister and Cheburek. Likes to talk with me, but I don't
+                        understand cat's language. Doesn't like to sit on people's lap or being picked up. Likes to
+                        put his face in sneackers.</p>
+                    </div>
+                </li>
+            </ul>
+        </section>     
+    </main>
+    <footer class="contacts">
+        <ul class="contacts__icons">
+            <li class="facebook_icon">
+                <a class="icon_link" href="https://www.facebook.com/profile.php?id=100053338170239">
+                        <i class="fa fa-facebook-official fa-3x" aria-hidden="true"></i>
+                </a>
+            </li>
+            <li class="icon_link" class="instagram_icon">
+                <a class="icon_link" href="https://www.instagram.com/jeiny_groove/">
+                    <i class="fa fa-instagram fa-3x" aria-hidden="true"></i>
+                </a>
+            </li>
+            <li class="icon_link" class="linkedin_icon">
+                <a class="icon_link" href="https://www.linkedin.com/in/olga-shimanskaia-988540195/">
+                    <i class="fa fa-linkedin-square fa-3x" aria-hidden="true"></i>
+                </a>
+            </li>
+            <li class="icon_link" class="github_icon">
+                <a class="icon_link" href="https://github.com/jeinygroove">
+                    <i class="fa fa-github-square fa-3x" aria-hidden="true"></i>
```

Comment:
Oh OK, so I see you're using font-awesome here. So maybe you can leave the contacts session to be done in a future pull request, to make this one shorter...